sample_factory/enjoy.py

```diff
@@ -47,16 +47,27 @@ def visualize_policy_inputs(normalized_obs: Dict[str, Tensor]) -> None:
     )  # this will be different frame-by-frame but probably good enough to give us an idea?
     scale = 5
     obs = cv2.resize(obs, (obs.shape[1] * scale, obs.shape[0] * scale), interpolation=cv2.INTER_NEAREST)
+
     cv2.imshow("policy inputs", obs)
+    cv2.waitKey(delay=1)
 
 
-def render_frame(cfg, env, video_frames, num_frames):
+def render_frame(cfg, env, video_frames, num_episodes, last_render_start):
     if cfg.save_video:
```

Comment:
Is there a reason this is an if-else function? Can we render on screen AND capture the video? 

... probably not since we need rgb_array, huh. We can show it with imshow but okay :)